CH-EDC Docker Image Build Job failing

# Bug Report

## Describe the Bug
The Docker image build job is failing
```
Run cd clearing-house-edc && docker build -t ghcr.io/truzzt/ids-basecamp-clearing/ch-edc:$DOCKER_IMAGE_TAG -f launchers/connector-prod/Dockerfile .
/home/runner/work/_temp/201d45b4-ec39-4c02-b344-e11eee1d64fa.sh: line 1: cd: clearing-house-edc: No such file or directory
Error: Process completed with exit code 1.
```

### Expected Behavior
The Docker image build job should publish a Docker image with the release version as tag.

--- a/.github/workflows/release-publish.yml
+++ b/.github/workflows/release-publish.yml
@@ -58,9 +58,6 @@ jobs:
     runs-on: ubuntu-latest
     needs: release
     if: ${{ needs.release.outputs.new_tag_version != '' }}
-    defaults:
-        run:
-          working-directory: ./clearing-house-edc
     permissions:
       contents: read
       packages: write

